### ISSUE TITLE
Windows 10 Pro>> will not start

### ISSUE THREAD

**Issue by rzw** [WikidPad/WikidPad#63]
I ran 2.3rc02 until recently. Wouldn't start today. Reinstalled for portable mode. No success. Temporarily disabled virus scanners. still no success. There is no error log. Just nothing happening

**Comment by vashnt707** [comment]
Can you post the log message? WikidPad_Error.log

**Comment by benkinooby** [comment]
@rzw I do not think that any resources or time will be spent on investigating your issue with 2.3rc02, for the following reasons:
* 2.4alpha01 is considered quite stable (to be fair, it should be versioned as such and lose its alpha status)
* you have a working WikidPad setup
* No error messages for 2.3rc0 provided
* limited developer resources/time available to investigate this issue

I am closing this issue, not because I think it's not relevant but because keeping it open will not have any beneficial effect or will lead to any targeted action. Hope you follow this reasoning.
